UbiquityOS-User-Manual/Introduction.md

```diff
@@ -0,0 +1,32 @@
+---
+coverY: 0
+---
+
+# Introduction
+
+## **Welcome to Ubiquity DAO**
+
+Ubiquity is a Decentralized Autonomous Organization (DAO) building products and services on Ethereum which leverage our flagship product, the Ubiquity Dollar (uAD). The DAO is committed to making the Ubiquity Dollar a cornerstone of the metaverse economy.
```

Comment:
Please replace all instances of uAD with UUSD.

We rebranded the stablecoin from "Ubiquity Algorithmic Dollar" to "Ubiquity Dollar" because algorithmic stablecoins have a bad reputation.